src/main/java/de/doubleslash/usb_led_matrix/view/SettingsView.java

```diff
@@ -0,0 +1,94 @@
+package de.doubleslash.usb_led_matrix.view;
+
+import de.doubleslash.usb_led_matrix.model.AvailabilityStatus;
+import javafx.fxml.FXML;
+import javafx.scene.control.ColorPicker;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.paint.Color;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class SettingsView {
+
+   @FXML
+   private ColorPicker AwayColor;
+
+   @FXML
+   private ColorPicker availableColor;
+
+   @FXML
+   private ColorPicker beRightBack;
+
+   @FXML
+   private ColorPicker busyColor;
+
+   @FXML
+   private ColorPicker doNotDisturbColor;
+
+   String filePath = "TeamsColor.properties";
+   File file = new File(filePath);
+
+   @FXML
+   public void initialize() throws IOException {
+      loadProperty();
+   }
+
+   @FXML
+   void saveButton(MouseEvent event) {
+      AvailabilityStatus.Away.setColor(AwayColor.getValue());
+      AvailabilityStatus.DoNotDisturb.setColor(doNotDisturbColor.getValue());
+      AvailabilityStatus.Busy.setColor(busyColor.getValue());
+      AvailabilityStatus.Available.setColor(availableColor.getValue());
+      AvailabilityStatus.BeRightBack.setColor((beRightBack.getValue()));
+
+      try (final FileWriter fw = new FileWriter("TeamsColor.properties")) {
+         final Properties p = new Properties();
+         p.setProperty("away", String.valueOf(AwayColor.getValue()));
+         p.setProperty("doNotDisturb", String.valueOf(doNotDisturbColor.getValue()));
+         p.setProperty("busy", String.valueOf(busyColor.getValue()));
+         p.setProperty("available", String.valueOf(availableColor.getValue()));
+         p.setProperty("beRightBack", String.valueOf(beRightBack.getValue()));
+         p.store(fw, "");
+      } catch (IOException e) {
+         throw new RuntimeException(e);
+      }
+      loadProperty();
+
+   }
+
+   public void loadProperty() {
+
+      if (file.exists()) {
+
+         final String fileName = "TeamsColor.properties";
+         try {
+            final InputStream input = new FileInputStream(new File(fileName));
```

Comment:
You can use [try-with-resources feature of Java](https://www.baeldung.com/java-try-with-resources) that will take care of closing File resources for you.